docs/rfcs/82-add-tee-support.md

```diff
@@ -19,6 +19,13 @@ I propose to add support for `tee` command to `BeyondCTL`. `BeyondCTL` will tee
 byctl tee [command options] [target]
 ```
 
+### Command options
+
+```
+--expect-size  expected size of the input file (default: "128MiB")
```

Comment:
Will we use `expected-size` or `expect-size`?

Reply:
We will use it for multipart upload, which requires partSize, the default is 128MiB.